packages/delegation-process/src/utilites/errorMappers.ts

```diff
@@ -36,11 +41,6 @@ export const createProducerDelegationErrorMapper = (
     )
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
-export const getDelegationErrorMapper = (error: ApiError<ErrorCodes>): number =>
-  match(error.code)
-    .with("delegationNotFound", () => HTTP_STATUS_NOT_FOUND)
```

Comment:
Before `getDelegationErrorMapper` was used in both `getDelegations` and `getDelegationById` routes.
Now it has been deleted in favor of two separated error mappers: `getDelegationByIdErrorMapper` which handles the  `delegationNotFound` and the `getDelegationsErrorMapper` which is an empty mapper since the service does not throw any specific errors